dynatademand/schemas/request/query/get_invoices_summary.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Is there no schema for this in the docs yet?

Reply:
I don't know what schema in this context means. Perhaps `startDate`, `endDate` and `extProjectId`?

Reply:
From the docs. For example, we got the query schema for get attributes from 

https://developers.dynata.com/demand-api-reference/data_endpoints/attributes/get-attributes

by clicking on the copy button at the right of "query parameters".

Reply:
The black thing atop the text here: 
<img width="126" alt="Screen Shot 2020-01-23 at 3 35 49 PM" src="https://user-images.githubusercontent.com/51792608/73025943-29c13b00-3df6-11ea-839a-c46d06ce17e1.png">

Reply:
Okay. Will read now.